.github/workflows/code_style_java.yml

```diff
@@ -0,0 +1,15 @@
+name: Code Style Java
+on: [push, pull_request]
+
+jobs:
+  InferenceEngine:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+
+      - name: Check java code style
+        run: |
+          inference-engine/ie_bridges/java/scripts/codestyle_check.sh
+          git diff >java_code_style_diff.patch
```

Comment:
`git diff` is a step which should be performed in case of failure of code style check